e2e-tests/README.md

```diff
@@ -48,34 +48,35 @@ ___Running tests on SauceLabs___
 
 To use SauceLabs instead of an instance of Firefox running locally, do the following:
 
-Create a text file called `.saucelabs` and put it in the e2e-tests directory. Get a username and key for SauceLabs and then add the following details to the `.saucelabs` file:
+Create a text file called `.saucelabs` and put it in your home directory. Get a
+username and key for SauceLabs and then add the following details to the
+`.saucelabs` file:
 
-	[credentials]
-	username = <SauceLabs user name>
-	key = <SauceLabs API key>
+```ini
+[credentials]
+username = <SauceLabs user name>
+key = <SauceLabs API key>
+```
 
-Then you can run the tests against staging using the following command
+Then you can run the tests against staging using the following command:
 
 	$ tox -e py27 -- --driver SauceLabs --capability browserName Firefox
 
-If you wish to run them against different environemts, set `PYTEST_BASE_URL` as indicated in the sections above for running tests against localhost or production
+If you wish to run them against different environments, set `PYTEST_BASE_URL`
+as indicated in the sections above for running tests against localhost or
+production.
 
 ___Running tests using headless Firefox___
 
-NOTE: These instructions are for Linux or macOS only.
+To run the tests using a copy of Firefox that can be run in 'headless' mode
+(meaning with no UI), do the following:
 
-To run the tests using a copy of Firefox that can be run in 'headless' mode (meaning with no UI), do the following:
-
-Set an environment variable `MOZ_HEADLESS` to be '1'. Check the documentation for your shell on how to do this. For zsh you can do using the following command:
+Set an environment variable `MOZ_HEADLESS` to be '1'. Check the documentation
+for your shell on how to do this. For zsh you can do using the following
+command:
 
 	$ export MOZ_HEADLESS=1
 
-Next you need to set an environment variable `MOZ_BINARY_PATH` to be the location of the binary of Firefox you want to use. For example, if you were using zsh and Firefox Nightly (which supports running in headless mode) on macOS:
-
-        $ export MOZ_BINARY_PATH='/Applications/FirefoxNightly.app/Contents/MacOS/firefox' 
```

Comment:
This is not a requirement for headless mode, and we already cover specifying a Firefox binary path earlier in the README.